base_custom_filter/models/ir_filters_group.py

```diff
@@ -0,0 +1,41 @@
+# Migrated to v14.0 by Ashish Hirpara (https://www.ashish-hirpara.com)
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class IrFiltersGroup(models.Model):
+    _name = "ir.filters.group"
+    _description = "Filters Group"
+    _order = "sequence, name, id"
+
+    def _selection_type(self):
+        return [("filter", "Filter"), ("groupby", "Group By")]
+
+    sequence = fields.Integer()
+    model_id = fields.Selection(
+        selection="_list_all_models", string="Model", required=True
```

Comment:
Can we remove the method from this module and instead use the standard method by adding an import of `from odoo.addons.base.models.ir_filters import IrFilters`?

```suggestion
        selection="IrFilters._list_all_models", string="Model", required=True
```